pkg/monitor/ptrace/ptrace.go

```diff
@@ -720,15 +724,18 @@ func onSyscallReturn(pid int, cstate *syscallState) error {
 
 ///////////////////////////////////
 
+func getIntParam(pid int, ptr uint64) int {
+	return int(int32(ptr))
+}
+
 func getStringParam(pid int, ptr uint64) string {
-	var out [256]byte
+	var out [4096]byte
```

Comment:
any problems with the original buffer size other than doing more than one iteration through the for loop?

Reply:
looks like i misread this. 4096 is the max size the value can be, but reading 256 at a time in a loop is fine. will revert.